### PR TITLE
Features/41

### PR DESCRIPTION
**Pull request by jmonterrubio**

### CODE REVIEW

**Review by dortegau — APPROVED** [review]

File: molecule.yml [hunk 1]
```diff
@@ -50,18 +53,18 @@ vagrant:
     - name: nginx.vm
       ansible_groups:
         - group01
-        - nginx
       interfaces:
         - network_name: private_network
-          type: dhcp
+          type: static
```
Comment (jnogol) [inline molecule.yml:58]: Just for curiosity, why is this needed?

**Review by jnogol — APPROVED** [review]

File: molecule.yml (hunk 1 above)
Comment (jmonterrubio) [reply molecule.yml:58]: This is because the vagrant-hostmanager plugin doesn't manage host in dhcp mode :(
